indico/modules/events/papers/controllers/management.py

```diff
@@ -16,22 +16,27 @@
 
 from __future__ import unicode_literals
 
-from flask import session, request, render_template
+from flask import session, request, render_template, flash, redirect
 
 from indico.modules.events.papers import logger
 from indico.modules.events.papers.controllers.base import RHManagePapersBase
-from indico.modules.events.papers.forms import PaperManagersForm
+from indico.modules.events.papers.forms import PaperManagersForm, PapersScheduleForm
 from indico.modules.events.papers.operations import set_reviewing_state
+from indico.modules.events.papers.settings import paper_reviewing_settings
 from indico.modules.events.papers.views import WPManagePapers
 from indico.modules.events.util import update_object_principals
+from indico.modules.events.papers.operations import schedule_cfp, open_cfp, close_cfp
+from indico.util.i18n import _
+from indico.web.flask.util import url_for
+from indico.web.forms.base import FormDefaults
 from indico.web.util import jsonify_data, jsonify_form
 
 
 def _render_paper_dashboard(event, view_class=None):
     if view_class:
-        return view_class.render_template('management/overview.html', event.as_legacy, event=event)
+        return view_class.render_template('management/overview.html', event.as_legacy, event=event, cfp=event.cfp)
```

Comment:
This information is already passed to the template. If you really want to use `cfp` instead of `event.cfp` in the template, you can simply do `{% set cfp = event.cfp %}` in the template file.